.travis.yml

```diff
@@ -86,8 +86,8 @@ before_install:
   fi;
 
 - travis_retry pip install -r requirements.txt
-- travis_retry pip install grabbit==0.1.2
-- travis_retry git clone -b 0.6.5 https://github.com/INCF/pybids.git ${HOME}/pybids && pip install -e ${HOME}/pybids
+- travis_retry pip install grabbit==0.2.6
+- travis_retry pip install git+https://github.com/bids-standard/pybids.git@0.7.0#egg=pybids
```

Comment:
Tests [skip](https://travis-ci.org/nipy/nipype/jobs/478502837#L6273-L6279) unless it's installed in editable mode.

```
interfaces/tests/test_io.py::test_bids_grabber 
[gw0] [ 83%] SKIPPED interfaces/tests/test_io.py::test_bids_grabber 
interfaces/tests/test_io.py::test_bids_fields 
[gw0] [ 84%] SKIPPED interfaces/tests/test_io.py::test_bids_fields 
interfaces/tests/test_io.py::test_bids_infields_outfields 
[gw0] [ 84%] SKIPPED interfaces/tests/test_io.py::test_bids_infields_outfields 
```

Reply:
That is by design because when you install in editable mode you get the data that is needed for the tests

Reply:
Right. So we need to switch back to editable.

Reply:
Oh, I thought it was editable, not sure why that changed.